### PR TITLE
feat(all): upgrade Angular to v10

### PR DESCRIPTION
**Pull request by griest024**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Fixes: N/A

## Does this PR introduce a breaking change?
```
[x] Yes
[ ] No
```

## Other information

### CODE REVIEW

**Review by lderrickable — CHANGES_REQUESTED** [review]

File: apps/demo/src/app/product/pages/product-grid-view/product-grid-view.component.spec.ts [hunk 1]
```diff
@@ -28,15 +19,14 @@ describe('ProductGridViewComponent', () => {
...
       imports: [
         DaffContainerModule,
         DaffLoadingIconModule,
-        ProductGridModule
-      ],
-      providers: [
-        { provide: DaffProductGridFacade, useClass: MockDaffProductGridFacade }
+        ProductGridModule,
+        DaffProductTestingModule,
+        DaffCartTestingModule
```
[the remark this answers is not in the document]
Comment (griest024) [reply apps/demo/src/app/product/pages/product-grid-view/product-grid-view.component.spec.ts]: good catch, removed

**Comment by lderrickable** [comment]
lgtm

**Review by lderrickable — APPROVED** [review]

**Comment by damienwebdev** [comment]
Merged manually.

**Pull request closed without merging**